src/main/java/org/jabref/gui/preferences/GeneralTab.java

```diff
@@ -138,6 +139,7 @@ public void setValues() {
         useOwner.setSelected(prefs.getBoolean(JabRefPreferences.USE_OWNER));
         overwriteOwner.setSelected(prefs.getBoolean(JabRefPreferences.OVERWRITE_OWNER));
         useTimeStamp.setSelected(prefs.getBoolean(JabRefPreferences.USE_TIME_STAMP));
+        useTimeStamp.getOnAction().handle(new ActionEvent()); // pretend we just clicked on the checkbox so it updates the UI
```

Comment:
I have difficulties understanding this, could you please explain why this is necessary?
Maybe there is a nice option.

Reply:
The actionEvent handler for useTimeStamp sets enable/disable on the other checkbox. The two straightforward options seemed, to me, to be

1. trigger the actionEvent handler
1. duplicate the enable/disable logic here in the setup method

I'm always suspicious of duplicating logic, since that's a pattern where bugs get introduced when the logic gets updated in one place but not in the other.

I suppose another alternative might be to extract the enable/disable logic into its own method, and then call that method from the actionEvent handler and from the setup method. Frankly, that's more clean, so I'd be happy to do that.